pytest_django/fixtures.py

```diff
@@ -182,7 +182,10 @@ def _set_suffix_to_test_databases(suffix):
             continue
 
         db_settings.setdefault("TEST", {})
-        db_settings["TEST"]["NAME"] = "{}_{}".format(test_name, suffix)
+        test_db_name = test_name
+        if not test_name.endswith("_{}".format(suffix)):
+            test_db_name = "{}_{}".format(test_name, suffix)
+        db_settings["TEST"]["NAME"] = test_db_name
```

Comment:
@SalmonMode 
Thanks.  I guess this makes sense in general, but here I also think that it would cover / work around some crucial problem.
When looking into `TestSqliteWithToxAndXdist.test_db_with_tox_suffix` the other day I've noticed that it is even flaky with pytest 5.3.3, i.e. not at least failing reliably.